From 8050eacf62790a969352020643d7b10e74c688fe Mon Sep 17 00:00:00 2001
From: Ciprian Marian Costea <ciprianmarian.costea@nxp.com>
Date: Thu, 11 Nov 2021 14:59:46 +0200
Subject: [PATCH 12/20] s32-gen1: pcie: doc: Document new dts binding entry

'no-check-serdes' dts entry is used in order to disable the check for
SerDes subsystem precence, which is performed by reading the MIDR2 register
from SIUL2_1 module.
This register is not accessible on current version of S32G3 SoC's. Hence
the need for this temporar dts entry.

Issue: ALB-8072
Upstream-Status: Pending 

Signed-off-by: Ciprian Marian Costea <ciprianmarian.costea@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 Documentation/devicetree/bindings/pci/fsl,s32-pcie.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/pci/fsl,s32-pcie.txt b/Documentation/devicetree/bindings/pci/fsl,s32-pcie.txt
index c6af5a5eca..d3d7f9e47b 100644
--- a/Documentation/devicetree/bindings/pci/fsl,s32-pcie.txt
+++ b/Documentation/devicetree/bindings/pci/fsl,s32-pcie.txt
@@ -18,6 +18,7 @@ Optional:
 - link-speed: Link speed (default 1 = Gen1);
 - ranges: Information used for address translation on the PCIe host bus
 	(see https://elinux.org/Device_Tree_Usage#PCI_Address_Translation)
+- no-check-serdes: Skip SerDes subsystem presence check
 
 Example:
 
-- 
2.17.1

